doc-content/enterprise-only/project-data/delete-branches-proc.adoc

```diff
@@ -7,6 +7,6 @@ You can delete any branch except for the *master* branch. {CENTRAL} does not all
 . Click image:project-data/dots.png[] in the upper-right corner of the screen and select *Delete Branch*.
 +
 .Deleting a branch menu
```

Comment:
Minor question - is the title "Deleting a branch menu" correct? I see the image file was renamed to `delete-branch-menu.png`.

Reply:
Sorted